docs/source/guides/feature_selection.ipynb

```diff
@@ -0,0 +1,375 @@
+    "\n",
+    "Featuretools enables users to perform feature selection on the results of Deep Feature Synthesis with three functions:\n",
+    "\n",
+    "- `ft.selection.remove_highly_null_features`\n",
+    "- `ft.selection.remove_single_value_features`\n",
+    "- `ft.selection.remove_highly_correlated_features`\n",
+    "\n",
+    "Each of the functions takes in a calculated feature matrix and can optionally take in the list of Feature definitions as well. If just a feature matrix is used as input, then just a feature matrix will be returned, but if both the matrix and the Feature definitions are used, then the results of feature selection will, similarly, be the matrix and the Feature definitions.\n",
+    "\n",
+    "We will describe each of these functions in depth, but first we must create an entity set with which we can run `ft.dfs`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import featuretools as ft\n",
+    "\n",
+    "from featuretools.selection import (\n",
+    "    remove_highly_correlated_features,\n",
+    "    remove_highly_null_features,\n",
+    "    remove_single_value_features,\n",
+    ")\n",
+    "\n",
+    "from featuretools.primitives import NaturalLanguage\n",
+    "from featuretools.demo.flight import load_flight\n",
+    "\n",
+    "es = load_flight(nrows=50)\n",
+    "es"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Remove Highly Null Features\n",
+    "\n",
+    "We might have a dataset with columns that have many null values. Deep Feature Synthesis might build features off of those null columns, creating even more highly null features. In this case, we might want to remove any features whose null values pass a certain threshold. Below is our feature matrix with such a case:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, features = ft.dfs(entityset=es,\n",
+    "                          target_entity=\"trip_logs\",\n",
+    "                          cutoff_time=pd.DataFrame({\n",
+    "                              'trip_log_id':[30, 1, 2, 3, 4],\n",
+    "                              'time':pd.to_datetime(['2016-09-22 00:00:00']*5)\n",
+    "                                                  }),\n",
+    "                          trans_primitives=[],\n",
+    "                          agg_primitives=[],\n",
+    "                          max_depth=2)\n",
+    "fm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We look at the above feature matrix and decide to remove the highly null features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ft.selection.remove_highly_null_features(fm)"
```

Comment:
in the other examples, it looks like we pass in `features=features`, would it make sense to do the same here?

Reply:
We definitely can! The feature list is optional, and it just impacts whether or not we get an updated feature list back, which I'd added in for the others so that we could highlight which features were removed.

If it's better to be consistent here, happy to pass the feature list in here

Reply:
actually, i dont know if consistency is what is most important. rather i would make sure it reads as a clear narrative. so it's fine to start without using the parameter, but then add it in later, just make sure it is explained

so, i'd make sure to clearly call out and explain the usage of the `features` parameter in the guide. perhaps the best way to do that would be a note section like we use elsewhere

![image](https://user-images.githubusercontent.com/371792/96377299-ea403900-1152-11eb-831c-3905c5ff4ca8.png)

Reply:
Got it--added a note when we first use features as a parameter and an extra line highlighting how we use the results

![image](https://user-images.githubusercontent.com/64278226/96467896-70af5600-11f9-11eb-9eb2-521ae4cbe520.png)

Reply:
looks good. thanks!